### PR TITLE
Color.color -> colors.colorize

### PR DESCRIPTION
**Pull request by RedHatter**
I apologize that this took me so long. I've been a bit busy lately.

All tests pass and I've changed all references I could find but I wouldn't be surprised if I missed something somewhere.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Does your submission pass tests?

> Note: please do NOT include build files (those generate by the build-xxx commands) with your PR,

Thank you for your help in advance, much appreciated !

### CODE REVIEW

**Comment by z3dev** [comment]
@RedHatter excellent! Feel free to ask @kaosat-dev or myself for a reveiw. ;)

File: packages/vtree/core/index.js [hunk 1]
```diff
@@ -82,9 +82,9 @@ const expand = (params, ...solids) => {
   return { children: solids, type: 'expand', params }
 }
 
-const color = (params, ...solids) => {
+const colors = (params, ...solids) => {
```
[the remark this answers is not in the document]
Comment (kaosat-dev) [reply packages/vtree/core/index.js:85]: yes, this is the same as the main api, so @z3dev is right, colorize it should be :)

File: packages/vtree/core/modeling/index-old.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 /* eslint-disable */
 const { toArray, flatten } = require('@jscad/array-utils')
 
-const color = (params, ...objects) => {
```
[the remark this answers is not in the document]
Comment (kaosat-dev) [reply packages/vtree/core/modeling/index-old.js:4]: no, no changes needed, I'll prep a PR with some cleanups

**Comment by z3dev** [comment]
@RedHatter back to you... just the one change it seems. :)

**Comment by z3dev** [comment]
@RedHatter back to you... just the one change it seems. :)

**Comment by z3dev** [comment]
@kaosat-dev it seems that @RedHatter has disappeared. do you mind if i merge this, and follow up with a small patch?

**Comment by z3dev** [comment]
Merging... will follow up with a patch for vtree.

**Pull request merged**